test/dummy/config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+EFBoshSTgD1qmTB2SGBrtRM4+HPQGC7cA9zfHVPgc0lhJ3R/R1HvMgERKO3A/a14XfJCMlm9B+U/WM+sonywUg59u5TO334LP+fCcTLu9FwvVE8OWBjG+cf2ITk3Hst8ytwgM7aFnBrzcNJQtNPBLehrICbUOogVD+FLTRTGg+gSr+QhYm31yXkvqG+p6U2Tw6xdBA9g2dz4ShF+fKOy1pw3YnzPlx4OGhUoYCtGgT7Uc9Sj3oBtRR9ZSI786QNVkhCYpCD2LeqHBrFHGfCNAmwLEwfwZGRh+uybl9YIKTp3f0qLkmQeVr3yuVksH8FW80lGea7ZU6qoq7QwBD2OrCLRt+Ex7m/MuH1e8Qz0rsTkaNxnS0gmT5RiEcf87NlipN5lg4+FBqrMk+NbGnsQuuOvDmIY--b9MngGf45VqeJNb/--Z8hYfzHT/7srWy4jyOHX9A==
```

Comment:
Any way to do this without this? Would be nice for others to be able to boot without needing secrets.

Reply:
@timirwin This was generated automatically..  I don't think it's required since I'm not using encrypted credentials anywhere.  Going to try removing it.